content/docs/examples/userauthentication.md

```diff
@@ -4,28 +4,52 @@
     "title": "Integrate User Authentication",
     "linktitle": "userauthentication",
     "card_icon": "ti-id-badge",
-    "card_body": "Learn how add User Authentication with Auth0",
+    "card_body": "Learn how to add User Authentication with Auth0",
     "weight": "2"
 }
 
 
 # Add User Authentication to Your Application
 {{<header_divider>}}
 
-User Authentication and Management is one of the most important aspects aspects in application development.
+User Authentication and Management is one of the most important aspects in application development.
 
 With the `Cdev API` resource, we can add granular [User Authorization](https://auth0.com/intro-to-iam/what-is-authorization/) and [Role-Based Access Control (RBAC)](https://auth0.com/docs/manage-users/access-control/rbac) to any backend. The `Cdev API` resource supports
-user authentication through JWT Tokens (need more details on the specifics).
+user authentication through [JSON Web Tokens(JWT)](https://jwt.io/introduction)(need more details on the specifics).
```

Comment:
You can make a tooltip with a link to https://auth0.com/docs/secure/tokens/json-web-tokens for more information about JWT's. You can add a simple message like 'You can learn more about JWT authentication at <link>`

Reply:
Should I omit line 18?

Reply:
Thanks!